examples/input.html

```diff
@@ -19,73 +21,74 @@
 
   <script>
 
-  var book = ePub();
-  var rendition;
+    var book = ePub();
```

Comment:
Looks like this PR has a lot formatting commits from the editor, which makes it really hard to read. 

I wonder if there's a way to revert them (since these changes probably wouldn't be accepted by the upstream repo), or if that's too hard. If we do keep them, could we do a PR that just touches these files to get the formatting changes, merge that in, and then have a PR that's only the changes we want for audio?